Add back link

deberny opened the pull request:
During user testing we have discovered a user need to go back to previous page.

Added back links at the end of raise repair pages. This gives the
user the ability to go back and edit their input.

Added improvements to booking call back time logic. We have decided that for picking a call back slots it's better if don't use "avoid school run ". Because in this scenario it's more likely that a person can answer the phone during that time. And this removes complexity.

Improve the wording to add more clarity.

Commit:
Ability to report repair if not on list - new views

Added new views for a scenario where user chooses 'something else' from
what is the problem list.

Commit:
Ability to go back to previous page - raising repair

Added back links at the end of raise repair pages. This gives the
user the ability to go back and edit their input.

Added improvements to booking call back time logic.

dgmstuart commented on app/views/raiserepair/book-appointment-contact.html:
```diff
@@ -30,13 +30,17 @@ <h2 class="heading-medium">
 
         <div class="form-group">
-          <input class="button" type="submit" value="Continue">
+          <input class="button" type="submit" value="Confirm and book a repair">
```
I'm not sure about the phrasing "book a repair"

Maybe "Confirm and book this appointment" makes more sense?

dgmstuart commented on app/views/raiserepair/describe-repair-address.html:
```diff
@@ -56,12 +56,17 @@ <h2 class="heading-medium">
       <aside role="submited-steps">
         <h3 class="heading-large"><span class="heading-secondary">Your report</span></h3>
         <p class="heading-small">Problem</p>
+        <p>Something else</p>
```
Might this be confusing to the resident? Do we need it?

dgmstuart commented on app/views/raiserepair/describe-repair-address.html:
```diff
@@ -74,16 +79,17 @@ <h3 class="heading-large"><span class="heading-secondary">Your report</span></h3
         <p>Tel number: {{ data['telephone-number'] }}</p>
         <p>Available for call:
 
-          {% if data['morningCall'] %}
```
As discussed, let's remove "school run"

deberny replied:
done 👍

dgmstuart commented on app/views/raiserepair/repair-details.html:
```diff
@@ -55,7 +55,19 @@ <h2 class="heading-medium">
 
       </form>
+      <p>
+        <a href="/raiserepair/what-is-problem/">Back to what do you need to be fixed</a>
```
Might 'what do you need to be fixed?' make more sense in quotes?

Commit:
Removed avoid school run from call back time slot

We have discussed the logic of choosing and showing call back
time slots and decided it's better to remove 'avoid school run'.
In this scenario it's more likely that a person can answer the phone
in more flexible time frame.

Removed 'Something else' from sidebar as it may confuse users.

Further refinement of wording for buttons and back links.

dgmstuart committed:
(fix) remove school run from 'something else' path

The previous commits went some way towards this, but left some
references lying around

dgmstuart reviewed (approved):
Latest changes look good - will merge
